cm_sdk/api/__init__.py

```diff
@@ -0,0 +1,4 @@
+__author__ = 'e0d'
+
+class APIException(Exception):
```

Comment:
It's specifically intended to be the most general class of exception for the API.